content/support/changelog/ap/index.md

```diff
@@ -14,9 +14,62 @@ title: Accounts Payable Changelog
 
 Monitor this page to keep up with the [Accounts Payable API]({{ stache.config.portal_endpoints_AP }}) latest changes and {{ stache.config.api_type_name }} service releases.
 
-## 2018-04-20
+## 2018-06-01
 
-### Changed
+### New
+
+Added the following endpoint:
+
+<div class="table-responsive">
+	<table class="table table-striped table-hover">
+		<thead>
+			<tr>
+				<th>Operation</th>
+				<th>Method</th>
+				<th>Route</th>
+			</tr>
+		</thead>
+		<tbody>
+			<tr class="clickable-row" data-url="{{ stache.config.portal_endpoints_invoice_edit_patch }}">
+				<td>Invoice edit</td>
+				<td>PATCH</td>
+				<td>/invoices/{invoice_id}</td>
+			</tr>
+		</tbody>
+	</table>
+</div>
+
+With the new endpoint, note the following:
+
+- You can edit the following values of an existing invoice:
```

Comment:
I think "values" should be "properties."

Reply:
Agreed.